src/AdminConsole/ApiHelper.cs

```diff
@@ -43,4 +43,26 @@ public async Task<NewAppResponse> CreateApplication(NewAppOptions registerOption
         res.EnsureSuccessStatusCode();
         return await res.Content.ReadFromJsonAsync<NewAppResponse>();
     }
+
+    public async Task<IEnumerable<string>> ListApplicationsPendingDeletionAsync()
+    {
+        try
+        {
+            var res = await _client.GetAsync("apps/list_pending_deletion");

```

Comment:
I think dashes are `-` preferred by the industry? But I don't have a strong opinion, `_` is fine for me.